src/Resources/Resources.Sdk/Readme.md

```diff
@@ -5,46 +5,138 @@ This directory contains management plane service clients of Az.Resources module.
 In this directory, run AutoRest:
 ```
 autorest --reset
```

Comment:
Each time, you update any packages, we will need to remove all the content in Generated folder. Otherwise, some content may not be updated accordingly since you set `clear-output-folder: false`. So please add a step here to remove all the content in Generated folder.
```
rm -r Generated/* 
```